plugins/UFPWriter/UFPWriter.py

```diff
@@ -60,5 +60,43 @@ def write(self, stream, nodes, mode = MeshWriter.OutputMode.BinaryMode):
         else:
             Logger.log("d", "Thumbnail not created, cannot save it")
 
+        #Store the material.
+        application = Application.getInstance()
+        machine_manager = application.getMachineManager()
+        global_stack = machine_manager.activeMachine
+        container_registry = application.getContainerRegistry()
+
+        archive.addContentType(extension="xml.fdm_material", mime_type="application/x-ultimaker-material-profile")
+
+        added_materials = []
+        for extruder_stack in global_stack.extruders.values():
+            material = extruder_stack.material
+            material_file_name = material.getMetaData()["base_file"] + ".xml.fdm_material"
+            material_file_name = "/Cura/" + material_file_name
```

Comment:
What does /Cura do here? (I know it's the (not final?) file path, but it still can use some inline documentation).